src/lib/components/sections/navbar.svelte

```diff
@@ -1,18 +1,23 @@
 <script lang="ts">
   import { page } from '$app/stores';
+  import LightMode from '../icons/light-mode.svelte';
+  import DarkMode from '../icons/dark-mode.svelte';
```

Comment:
For the sake of consistency, I think we should import the icons from our internal `/$lib/` folder.

```suggestion
  import LightMode from '$lib/components/icons/light-mode.svelte';
  import DarkMode from '$lib/components/icons/dark-mode.svelte';
```

Reply:
Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/401/commits/084501e0c46803937cc23544bf3c502ff10d47a2>.